extensions/authentication-jwt/package.json

```diff
@@ -21,21 +21,21 @@
     "access": "public"
   },
   "dependencies": {
-    "@loopback/authentication": "^4.2.9",
-    "@loopback/core": "^2.9.1",
-    "@loopback/rest": "^5.2.0",
     "@loopback/rest-explorer": "^2.2.6",
```

Comment:
While we are reviewing dependencies, is it intentional that the extension `@loopback/authentication-jwt` installs `@loopback/rest-explorer` as a dependency? Shouldn't the application developer decide if they want to use API Explorer or not?

Reply:
+1 to remove it.

Reply:
AFAICT, API Explorer is used by a test application, see 

https://github.com/strongloop/loopback-next/blob/cb9e6d79d4db265f39de551f81a824a5cb5c3dcb/extensions/authentication-jwt/src/__tests__/fixtures/application.ts#L38

I am going to move it to `devDependencies` - see https://github.com/strongloop/loopback-next/pull/6009